SPECS/libnetfilter_conntrack/libnetfilter_conntrack.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "libnetfilter_conntrack-1.0.8.tar.bz2": "0cd13be008923528687af6c6b860f35392d49251c04ee0648282d36b1faec1cf"
+  "libnetfilter_conntrack-1.0.9.tar.bz2": "67bd9df49fe34e8b82144f6dfb93b320f384a8ea59727e92ff8d18b5f4b579a8"
  }
 }
```

Comment:
EOF Newline
```suggestion
}

```

Reply:
While it would be nice to add EOF newline, this would be an unnecessary edit for this stable release change.